setup-env

```diff
@@ -27,7 +27,7 @@ export EV3DEV_TOOLCHAIN="/opt/arm-2011.03/bin"
 export EV3DEV_ABI="arm-none-eabi-"
 
 # This is where the scripts put all of generated files
-export EV3DEV_BUILD_AREA="$HOME/build-area"
+export EV3DEV_BUILD_AREA="$(pwd)/build-area"
```

Comment:
The `$HOME/build-area` here is actually on purpose. There are other tools that also use this directory by default, so I was trying to keep everything in one place.

Of course, I could just override it locally for me. I'm starting to think they way you have it may be better for most people.